docs-2.0/3.ngql-guide/11.edge-statements/2.update-edge.md

```diff
@@ -0,0 +1,24 @@
+# UPDATE EDGE
+
+```ngql
+UPDATE EDGE <src_vid> -> <dest_vid> [@rank] OF <edge_type> SET <update_columns> [WHEN <condition>] [YIELD <columns>]
```

Comment:
can we replace "columns" with "properties" in this statment? 

In Nebula Graph, "property" should be emphasized?